### PR TITLE
rename "toepassing" to "applicatie" for Dutch strings

### PR DESCRIPTION
**Pull request by RichieB2B**
Nobody uses the Dutch word "toepassing" in the context of "application" anymore. It confused me quite a bit.

### CODE REVIEW

**Comment by michaelherger** [comment]
Oh, here we go 😁. Thanks!

File: strings.txt [hunk 1]
```diff
@@ -20991,7 +20991,7 @@ SETUP_DEBUG_SERVER_LOG_DESC
 	FI	Logitech Media Server pitää lokitiedostoa kaikista sovelluksiin liittyvistä toiminnoista (äänen virtautus, infrapuna jne.) täällä:
 	FR	Le Logitech Media Server conserve un fichier journal de toutes les activités liées à l'application (diffusion audio, infrarouge, etc.), à l'emplacement suivant :
 	IT	Logitech Media Server mantiene un file di registro per tutte le attività relative alle applicazioni (stream audio, raggi infrarossi e così via) nel seguente percorso:
-	NL	Logitech Media Server bewaart hier een logboek voor alle toepassingsactiviteiten (audio-streaming, infrarood, enz.):
+	NL	Logitech Media Server bewaart hier een logboek voor alle applicatie-activiteiten (audio-streaming, infrarood, enz.):
```
Comment (terual) [inline strings.txt:20994]: applicatieactiviteiten is without a dash, please remove the dash.

File: strings.txt [hunk 2]
```diff
@@ -21906,7 +21906,7 @@ PERSIST_DEBUG_SETTINGS
 	FI	Tallenna kirjausasetukset käytettäväksi, kun sovellus käynnistetään seuraavaksi uudelleen
 	FR	Enregistrer les réglages de journalisation pour une utilisation au prochain démarrage de l'application
 	IT	Salva e utilizza le impostazioni di registrazione al successivo avvio dell'applicazione
-	NL	Log-instellingen opslaan voor gebruik bij volgende herstart van toepassing
+	NL	Log-instellingen opslaan voor gebruik bij volgende applicatie herstart
```
Comment (terual) [inline strings.txt:21909]: applicatieherstart is one word, please remove the space between words.

**Comment by mherger** [comment]
Thanks @terual!

File: strings.txt (hunk 1 above)
Comment (RichieB2B) [reply strings.txt:20994]: Thanks for the review. Do I need to add the fix to this PR?

**Comment by mherger** [comment]
I already modified those two strings.

**Comment by terual** [comment]
@mherger, thanks!

**Pull request merged**